src/main/java/org/systemexception/lunarlander/main/Main.java

```diff
@@ -0,0 +1,142 @@
+package org.systemexception.lunarlander.main;
+
+import org.jbox2d.common.Vec2;
+import org.jbox2d.dynamics.Body;
+import org.lwjgl.LWJGLException;
+import org.lwjgl.opengl.Display;
+import org.lwjgl.opengl.DisplayMode;
+import org.lwjgl.opengl.GL11;
+import org.newdawn.slick.Color;
+import org.newdawn.slick.TrueTypeFont;
+import org.newdawn.slick.opengl.TextureImpl;
+import org.newdawn.slick.util.ResourceLoader;
+import org.systemexception.lunarlander.constants.BodiesNames;
+import org.systemexception.lunarlander.physics.GameEngine;
+
+import java.awt.*;
+import java.io.InputStream;
+
+import static org.lwjgl.opengl.GL11.*;
+
+/**
+ * @author leo
+ * @date 29/12/15 14:22
+ */
+public class Main {
+
+	private static final String WINDOW_TITLE = "Physics in 2D!";
+	private static final int[] WINDOW_DIMENSIONS = {800, 600};
+
+	private final static double TWO_PI = 2 * Math.PI;
+
+	private TrueTypeFont font;
+	private static GameEngine gameEngine;
+
+	public static void main(String[] args) {
+
+		Main main = new Main();
+		gameEngine = new GameEngine();
+		main.start();
+	}
+
+	private void start() {
+		initGL();
+		initFonts();
+		gameEngine.setUpObjects();
+
+		while (!Display.isCloseRequested()) {
+			GL11.glClear(GL11.GL_COLOR_BUFFER_BIT);
+			render();
+			gameEngine.input();
+			gameEngine.logic();
+
+			Display.update();
+			Display.sync(60);
+		}
+		Display.destroy();
+		System.exit(0);
+	}
+
+	private void initGL() {
+		try {
+			Display.setDisplayMode(new DisplayMode(WINDOW_DIMENSIONS[0], WINDOW_DIMENSIONS[1]));
+			Display.setTitle(WINDOW_TITLE);
+			Display.setVSyncEnabled(true);
+			Display.create();
+		} catch (LWJGLException e) {
+			e.printStackTrace();
+			System.exit(0);
+		}
+
+		glMatrixMode(GL_PROJECTION);
+
+		glShadeModel(GL11.GL_SMOOTH);
+		glDisable(GL11.GL_TEXTURE_2D);
+		glDisable(GL11.GL_DEPTH_TEST);
+		glDisable(GL11.GL_LIGHTING);
+
+		glClearColor(0.0f, 0.0f, 0.0f, 0.0f);
+		glClearDepth(1);
+
+		glEnable(GL11.GL_BLEND);
+		glBlendFunc(GL11.GL_SRC_ALPHA, GL11.GL_ONE_MINUS_SRC_ALPHA);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'GL11.GL_SRC_ALPHA' due to existing import 'org.lwjgl.opengl.GL11'](https://www.codacy.com/app/leo_4/lunar-lander/file/1692129201/issues/source?bid=3040650&fileBranchId=3043231#l82)